solr/core/src/test/org/apache/solr/handler/ReplicationTestHelper.java

```diff
@@ -62,14 +61,9 @@ public final class ReplicationTestHelper {
           + FileSystems.getDefault().getSeparator();
 
   public static JettySolrRunner createAndStartJetty(SolrInstance instance) throws Exception {
-    Files.copy(
-        SolrTestCaseJ4.TEST_HOME().resolve("solr.xml"),
-        Path.of(instance.getHomeDir(), "solr.xml"),
-        StandardCopyOption.REPLACE_EXISTING);
-    Properties nodeProperties = new Properties();
-    nodeProperties.setProperty("solr.data.dir", instance.getDataDir());
```

Comment:
I guess generally you have identified that setting solr.data.dir isn't necessary?